src/category_theory/limits/preserves/shapes/equalizers.lean

```diff
@@ -0,0 +1,104 @@
+/-
+Copyright (c) 2020 Bhavik Mehta. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Bhavik Mehta
+-/
+import category_theory.limits.shapes.equalizers
+import category_theory.limits.preserves.basic
+
+/-!
+# Preserving equalizers
+
+Constructions to relate the notions of preserving equalizers and reflecting equalizers
+to concrete forks.
+
+In particular, we show that `equalizer_comparison G f` is an isomorphism iff `G` preserves
+the limit of `f`.
+-/
+
+noncomputable theory
+
+universes v u₁ u₂
+
+open category_theory category_theory.category category_theory.limits
+
+variables {C : Type u₁} [category.{v} C]
+variables {D : Type u₂} [category.{v} D]
+variables (G : C ⥤ D)
+
+namespace preserves_equalizer
+
+variables {X Y Z : C} {f g : X ⟶ Y} {h : Z ⟶ X} (w : h ≫ f = h ≫ g)
+
+/--
+The map of a fork is a limit iff the fork consisting of the mapped morphisms is a limit. This
+essentially lets us commute `fork.of_ι` with `functor.map_cone`.
+-/
+def fork_map_cone_limit :
+  is_limit (G.map_cone (fork.of_ι h w)) ≃
+  is_limit (fork.of_ι (G.map h) (by simp only [←G.map_comp, w]) : fork (G.map f) (G.map g)) :=
+(is_limit.postcompose_hom_equiv (diagram_iso_parallel_pair _) _).symm.trans
+  (is_limit.equiv_iso_limit (fork.ext (iso.refl _) (by simp [fork.ι_eq_app_zero])))
+
+/-- The property of preserving equalizers expressed in terms of forks. -/
```

Comment:
Idem dito

Reply:
How about `is_limit_fork_map_of_is_limit`

Reply:
Do we not want `preserves` mentioned in there somewhere? If not, should the next one be `is_limit_of_is_limit_fork_map` as well?

Reply:
I think you're right, the `map` is enough instead of the preserves, amended!